### PR TITLE
Maintenance guidelines

### PR DESCRIPTION
**Pull request by lpil**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

Guidelines on how to maintain a project at Embark.

This are the guidelines from Notion, so hopefully you've already reviewed them 😉 

### Related Issues
https://github.com/EmbarkStudios/opensource/issues/2

### CODE REVIEW

File: content/maintenance-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+If a pull request is in an unfinished and un-mergeable state, and the
+contributor has not made any updates or indicated that they are currently
+working on it for over 2 weeks, close the pull request. Closing stale pull
+requests makes it clear the project is maintained, and makes it easier to
+tell the current state of work on the project.
+
+When closing leave a message thanking them for their contribution and
+inviting them to reopen the pull request when they have further changes.
+
+## Perform hand overs
```
Comment (arirawr) [inline content/maintenance-guidelines.md:94]: This should be hand-overs maybe?

File: content/maintenance-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+If you are unable to maintain the project for a period of time, e.g. if you
+are going on holiday, find another person at Embark to temporarily take over
+your responsibilities. Add to this person as a code owner to the repository
```
Comment (arirawr) [inline content/maintenance-guidelines.md:98]: Link to GitHub article about code owners.

File: content/maintenance-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+## Get help from the ecosystem team
+
+If you find yourself in a situation where you’re not sure what to do, reach
+out to the ecosystem team for support. They can be reached via the
+`#arkadia-ecosystem` channel in Slack.
+
+## Adhere to the code of conduct
+
+All open source Embark projects have a code of conduct which details the
```
Comment (arirawr) [inline content/maintenance-guidelines.md:115]: Link to Code of Conduct

File: content/maintenance-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+# Open source maintenance guidelines
+
```
Comment (arirawr) [inline content/maintenance-guidelines.md:2]: Maybe we can give a short foreword here. Just a couple sentences of context for folks who are linked directly into this page.

**Comment by lpil** [comment]
Great suggestions, I'll do that now

**Review by arirawr — APPROVED** [review]

**Comment by lpil** [comment]
Thank you!

**Pull request merged**